google-cloud-automl/CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Release History
 
+### 0.2.0 / 2019-08-14
+
+#### Features
+
+* Deprecate ColumnSpec#top_correlated_columns
+  * Deprecate ColumnSpec#top_correlated_columns
+  * Add BatchPredictResult#metadata.
+  * Add ConfusionMatrix#display_name
+  * Add TableSpec#valid_row_count
+  * Update documentation
+* Update documentation
+
 ### 0.1.0 / 2019-07-15
 
 * Initial release.
-
```

Comment:
Our accepted style guide is to have a newline at the end of files. Can this be maintained?